packages/peregrine/lib/context/cart.js

```diff
@@ -55,20 +55,16 @@ const CartContextProvider = props => {
         derivedCartState
     ]);
 
-    const apolloClient = useApolloClient();
     const [fetchCartId] = useMutation(CREATE_CART_MUTATION);
     const fetchCartDetails = useAwaitQuery(CART_DETAILS_QUERY);
 
     useEffect(() => {
-        // cartApi.getCartDetails initializes the cart if there isn't one. Also, we pass
-        // apolloClient to wipe the store in event of auth token expiry which
-        // will only happen if the user refreshes.
+        // cartApi.getCartDetails initializes the cart if there isn't one.
         cartApi.getCartDetails({
-            apolloClient,
```

Comment:
Passing things that can't be serialized, such as functions or objects containing functions, in action payloads is an anti-pattern in Redux. I understand the thought behind putting `apolloClient` in the payload, but we shouldn't have been doing this.